src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryCustomObject.cs

```diff
@@ -124,12 +157,55 @@ public static FormulaValue Index_CO(IRContext irContext, FormulaValue[] args)
 
         public static FormulaValue Value_CO(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, CustomObjectValue[] args)
```

Comment:
>Value_CO [](http://example.com/codeflow?start=35&length=8)

This should chain to the existing Value() func
- because there are a lot of corner cases (like % handling)
- to handle format args. 

What about:

FormulaValue fv = GetAsFv(impl);
return Value(runner, symbolContext, irContext,  
   new FV[] { fv, arg1 })


And then GetAsFv() has this switch logic and can be reused by the other converter functions.

Reply:
This should be a topic of discussion for a future PR.